framework/src/main/java/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -482,12 +502,18 @@ public Boolean visitDeclared_Declared(
   }
 
   /**
-   * A helper class for visitDeclared_Declared. There are subtypes of DefaultTypeHierarchy that need
-   * to customize the handling of type arguments. This method provides a convenient extension point.
+   * Checks that the type arguments in {@code supertype} contain the type arguments in {@code
```

Comment:
"Checks that" could mean that the method throws an exception or reports an error.
Or, "checks that" could indicate the method's return value.
Could you resolve the ambiguity?

I think that all the visit methods are returning a boolean value indicating whether the method's arguments are in a subtyping relationship.  It would be good to document that in the class Javadoc.